requirements.txt

```diff
@@ -1,2 +1,5 @@
 flake8
-pytfa
\ No newline at end of file
+pytfa
+#INCA_tools
```

Comment:
I think pytfa installs numpy and pandas. Since there is going to be more requirements in this repo I propose to specify the version of pandas and numpy if you think that it is necessary. Cobra sometimes has incompatibilities with wrong version of numpy or pandas.